bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,259 @@ class CoffeeLintBear:
+                        no_backticks: bool=True,
+                        no_implicit_braces: bool=True,
+                        no_function_call_without_parentheses: bool=True,
+                        no_interpolation_in_single_quotes: bool=True,
+                        no_stand_alone_at: bool=True,
+                        no_throwing_strings: bool=True,
+                        no_trailing_semicolons: bool=True,
+                        no_trailing_whitespaces: bool=True,
+                        apply_no_trailing_whitespaces_in_comments: bool=True,
+                        apply_no_trailing_whitespaces_in_empty_line: bool=True,
+                        no_unnecessary_double_quotes: bool=True,
+                        use_english_operator: bool=False,
+                        space_around_operators: bool=True,
+                        space_after_comma: bool=True,
+                        cyclomatic_complexity: bool=True,
+                        cyclomatic_complexity_value: int=10,
+                        duplicate_keys: bool=True,
+                        enforce_parentheses_on_empty_constructor: bool=True,
+                        line_endings: bool=False,
+                        line_endings_option: str='unix',
+                        no_this: bool=False,
+                        no_plusplus: bool=False,
+                        no_empty_parameter_list: bool=False,
+                        no_empty_functions: bool=False):
+        """
+        :param: max_line_length:
+            Maximum number of characters per line.
+        :param: apply_max_line_length_on_comments:
+            Determines if ``max_line_length`` should also affects comments or
+            not.
+        :param: space_before_and_after_arrow:
+            Determines if spaces should be used before and/or after the arrow.
+        :param: braces_spacing:
+            Checks if the proper spacing is used inside curly braces.
+        :param: braces_spacing_width:
+            Determines the number of white space after the opening ``{`` and
+            before the closing brace ``}`` given that there is something within
+            the braces.
+        :param: spacing_in_empty_braces:
+            Determines the number of white space after the opening ``{`` and
+            before the closing brace ``}`` given an empty content.
+        :param: camelcase_classes_name:
+            Checks whether the classes name should are in camel-case or not.
+        :param: spaces_before_and_after_colon:
+            Determines if spaces should be added before and after colons.
+        :param: spaces_before_colon:
+            Determines the number of space before colon when
+            ``spaces_before_and_after_colon`` is `` True``.
+        :param: spaces_after_colon:
+            Determines the number of space after colon when
+            ``spaces_before_and_after_colon`` is `` True``.
+        :param: end_file_with_a_single_newline:
```

Comment:
i don't really like its name, it sounds as if it forces an endline, while it checks for it. let's think about a better name.

hm: ``check_file_trailing_newline`` maybe? or ``check_whether_file_ends_in_newline`` ? what do you say?